### DISCUSSION TITLE
ABC Australia (Pay TV Channel Stream)

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#794] [Show and tell]
### Channel Name

ABC Australia Asia

### Channel ID

ABCAustralia.au

### Website

http://www.abcaustralia.net.au/

### Stream URL

http://203.154.177.124:8899/live/ch8/chunks.m3u8

### Notes

The link can only work on m3u8 apps. What about a normal m3u8? NOTE: "ABC Australia" logo on the down right side:
![abc-australia-logo-F32016E939-seeklogo com](https://user-images.githubusercontent.com/106740926/171972916-84adfac1-1762-4314-bb7a-bd80ab8da981.png)

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by darrencycheung** [comment]
I had found so many related m3u8 links. But they can't work. Here's the photo of the TV:
![Australia-IPTV-Lists-1024x576](https://user-images.githubusercontent.com/106740926/171973628-53d45885-66ee-4c2e-9687-1f1fc6bc5a3d.jpeg)

**Comment by darrencycheung** [comment]
https://en.wikipedia.org/wiki/ABC_Australia_(Southeast_Asian_TV_channel)
This TV is available on HK's Now TV Ch. 561 and ICable's Ch. 326.

**Comment by darrencycheung** [comment]
https://www.youtube.com/c/abcaustralia/featured
Former names: ABC Asia Pacific, Australia Network and Australia Plus. Today's name was introduced on July 1, 2018. Owned by ABC Australian Broadcasting Company.

**Comment by F640** [comment]
> http://203.154.177.124:8899/live/ch8/chunks.m3u8

Try setting referer to `http://sportsevent.xyz` (assuming that watermark in your stream sceenshot is the origin of URL). Recently tested with Kodi and it works (720p)

![IMG_20220604_120953](https://user-images.githubusercontent.com/92260436/171984979-185f5ea0-1d85-4bee-95ed-782c891c12e5.jpg)

> The link can only work on m3u8 apps. What about a normal m3u8?

Which one exactly? It can get worked by setting proper user agent and/or referrer and/or header actually, except if it's tokenized which will be hard. Some links refuse to play or redirected to specific stream if required settings didn't full filled.

I have one link to this channel, but it's protected with Widevine (DRM) and not in HD unlike yours. ~It's mpd actually~
![IMG_20220604_114659](https://user-images.githubusercontent.com/92260436/171983585-8443e681-f677-4889-a779-70f653ac8f22.jpg)

**Comment by darrencycheung** [comment]
OK. I'd discovered so many ABC Australia Plus live links but they can't working. I hope there will be a normal live m3u8 link of Australia Plus. So I can let the iptv github to add the link to the au playlist.

**Comment by darrencycheung** [comment]
And what about the link you said?

**Comment by darrencycheung** [comment]
I like this channel

**Comment by darrencycheung** [comment]
This channel doesn't have sound?!

**Comment by F640** [comment]
> This channel doesn't have sound?!

It has sound from my side. Try playing it again for a while, who knows that's just bug from your side.

> And what about the link you said?

Are you sure you won't be disappointed? My URL can only played with Widevine-capable IPTV player (like Kodi with IPTV PVR Simple and inputsream.adaptive add-on, not VLC), and it's not HD. If you're sure about that then i can give you.

**Comment by darrencycheung** [comment]
Ok. It doesn't matter to me

**Comment by darrencycheung** [comment]
Give the link to me please

**Comment by darrencycheung** [comment]
Sure

**Comment by F640** [comment]
License URL: https://mrpw.ptmnc01.verspective.net/?deviceId=NDIzMDJhZmUtYWRjMi0zNGJkLTkyN2EtYmE1ZDFlZWIwODEz
mpd: https://liveanevia.mncnow.id/live/eds/AustraliaPlus/sa_dash_vmx/AustraliaPlus.mpd

https://github.com/iptv-org/iptv/discussions/3840#discussioncomment-1162138
This is getting off-topic now

**Comment by darrencycheung** [comment]
Ok. I'll try it.

**Comment by darrencycheung** [comment]
I've discovered the mpd you said.

**Comment by darrencycheung** [comment]
What about m3u8 link?

**Comment by F640** [comment]
> What about m3u8 link?

It has m3u8 version, but protected with Apple FairPlay (another DRM), and i have no idea how to play that yet. Removing `vmx` suffix from both mpd and m3u8 will return internal server error (error 500).

**Comment by darrencycheung** [comment]
![Screenshot_2022-06-05-13-13-18-791_com batix app onlineplayer](https://user-images.githubusercontent.com/106740926/172036082-37f9d06a-41a5-4e90-9053-162155133c2a.jpg)
This is ch8 m3u8

**Comment by darrencycheung** [comment]
This is the Chinese TV app ver
![Screenshot_2022-06-14-19-55-38-105_com cloudtv](https://user-images.githubusercontent.com/106740926/173571513-840a3268-f933-4abf-ad85-798dbe9e9184.jpg)

**Comment by darrencycheung** [comment]
https://cdngb196.useetv.com/joss/133/abcaustralia/chunklist_w302187478_b684100_sleng.m3u8?enc=9-7OPAOrUXLLRG8VGh7kUg&uid=&exp=1656299585

**Comment by freearhey** [comment]
http://203.154.177.124:8899/live/ch8/chunks.m3u8 (offline)
<img width="365" alt="image" src="https://user-images.githubusercontent.com/7253922/232227604-723b0d5c-2afe-446a-8319-27f9b8045bc4.png">

https://cdngb196.useetv.com/joss/133/abcaustralia/chunklist_w302187478_b684100_sleng.m3u8?enc=9-7OPAOrUXLLRG8VGh7kUg&uid=&exp=1656299585 (offline)
<img width="837" alt="image" src="https://user-images.githubusercontent.com/7253922/232227675-c388a8ba-08d5-4c28-b05a-14312345c2d7.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19790
